src/mlpack/methods/ann/layer/base_layer.hpp

```diff
@@ -232,6 +233,17 @@ template <
 using GELUFunctionLayer = BaseLayer<
     ActivationFunction, InputDataType, OutputDataType>;
 
+/**
```

Comment:
Could you add this to the list at line 38

Reply:
Listed all the standard layers to the list. Thanks again.